src/types/io.ts

```diff
@@ -337,23 +380,14 @@ export interface AoIORead {
   }>;
   getTokenSupply(): Promise<AoTokenSupplyData>;
   getEpochSettings(params?: EpochInput): Promise<AoEpochSettings>;
-  getGateway({
-    address,
-  }: {
-    address: WalletAddress;
-  }): Promise<AoGateway | undefined>;
+  getGateway({ address }: AoAddressParams): Promise<AoGateway | undefined>;
   // TODO: these could be moved to a separate Gateways class that implements gateway specific interactions
   getGatewayDelegates({
     address,
     ...pageParams
-  }: {
-    address: WalletAddress;
-  } & PaginationParams<AoGatewayDelegateWithAddress>): Promise<
+  }: AoAddressParams & PaginationParams<AoGatewayDelegateWithAddress>): Promise<
     PaginationResult<AoGatewayDelegateWithAddress>
   >;
-  getGatewayDelegateAllowList(
```

Comment:
this seems like a duplicate of `getAllowedDelegates`. I can re-add this because of backwards compatibility, but not sure which to use as their definitions were slightly different. so leaving both is confusing for an integrator